src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -168,6 +168,7 @@ protected String getRecipeList(HttpServletRequest request, HttpServletResponse r
 
     boolean isTagQuery = (tagIDs != null && tagIDs.length > 0 && !tagIDs[0].equals("None"));
     boolean isCreatorQuery = (creatorToken != null && !creatorToken.equals("None"));
+    boolean isFollowedTagsQuery = (isCreatorQuery && sortingMethod == SortingMethod.TAGS);
 
```

Comment:
What if the user wants to sort by new or top for their custom feed? I feel like we should have some other flag to determine whether it's a followedTagsQuery